### DISCUSSION TITLE
grep-friendly --list general switch

### DISCUSSION THREAD

**Discussion by eadmaster** [uutils/coreutils#6249] [Ideas]
busybox has a`--list` switch to get the list of all supported commands, can the same be added to coreutils?

on windows i use it like this:
````
	FOR /F %%C IN ('call "%~dp0busybox.bat" --list') DO (
		IF [%TOOL%]==[%%C] (
			call "%~dp0busybox.bat" %*
			goto :eof
		)
	)
````

**Comment by tertsdiepraam** [comment]
Sounds like an excellent idea! We print the list on errors already so the necessary info is in the code already, so it should be easy.

**Reply by tertsdiepraam** [reply]
I made an [issue](https://github.com/uutils/coreutils/issues/6250) and a [pull request](https://github.com/uutils/coreutils/pull/6251) for this
